examples/sms_handler_demo.py

```diff
@@ -18,7 +18,9 @@
 from gsmmodem.modem import GsmModem
 
 def handleSms(sms):
-    print(u'== SMS message received ==\nFrom: {0}\nTime: {1}\nMessage:\n{2}\n'.format(sms.number, sms.time, sms.text))
+    # long sms Concatenation support: reference, parts, number
+    concat = sms.concat.__dict__ if sms.concat else {}
```

Comment:
Is it necessary to have this line here?
Could you move this logic to the module?
Just thinking.

Reply:
this process just for print purpose, 
  in GsmModem,  the `sms.concat` property will be:
  - dict (for long sms) 
  - None (for normal sms)

I think `None` is more clear than `{}` for normal sms 's `concat` info,  

what do you think about it?